packages/amplication-server/src/core/resource/resource.service.ts

```diff
@@ -596,12 +596,7 @@ export class ResourceService {
           authProvider: EnumAuthProviderType.Jwt, //@todo: remove this property
         },
 
-        gitRepository: {
-          isOverrideGitRepository: false,
-          name: "",
-          resourceId: "",
-          gitOrganizationId: "",
-        },
+        gitRepository: null,
```

Comment:
This type of change caused some bugs in the past - can you elaborate on the need for this change?

Reply:
This is indeed a very delicate place, and I would be happy to revisit it one day.
Because I had a few issues in this area, I copied the behavior of the wizard from the client - and null is the behavior we need as far as I understand.
I must say that it was not that easy to track that